.travis.yml

```diff
@@ -46,7 +52,7 @@ before_script:
   - |
     if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
       composer global require wp-coding-standards/wpcs
-      phpcs --config-set installed_paths $HOME/.composer/vendor/wp-coding-standards/wpcs
+      composer run lint:php:config
```

Comment:
This would require programmatically modifying `composer.json`, which I don't want to do.